lib/std/Target.zig

```diff
@@ -1430,6 +1444,7 @@ pub const Cpu = struct {
             avr,
             bpf,
             csky,
+            ez80,
```

Comment:
It's my understanding that the eZ80 is a variant of the Z80 that's binary compatible. If that is the case, the `Arch` tag should be `z80`, and any minor differences should be handled with feature bits for the eZ80.

Reply:
Actually, on further reading, it seems like the eZ80 has a larger address space than the Z80? In that case, a distinct `Arch` tag is justifiable, I think. We'd want a `z80` tag for targeting the older model with a smaller address space, but that doesn't have to be part of this PR.